WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "2.0.0-beta.4"
+  s.version       = "2.0.0-beta.5"
```

Comment:
Alas, through no fault of your own, I think we'll need to update the minor version & reset the beta counter (e.g., `2.1.0-beta.1`).